modules/data-model/subjects/impl/src/main/java/io/uhndata/cards/subjects/internal/serialize/SimpleSubjectProcessor.java

```diff
@@ -45,6 +45,13 @@ public String getName()
         return "simple";
     }
 
+    @Override
+    public String getDescription()
+    {
+        return "Only runs on Subjects.\n"
+            + "Simplify form serialization by removing child jcr properties.";
```

Comment:
"child jcr properties" makes me think of "descendent nodes' jcr properties". If that's not what is intended, drop "child"
```suggestion
            + "Simplify form serialization by removing the jcr properties.";
```